packages/react-router/lib/hooks.tsx

```diff
@@ -1046,14 +1046,14 @@ export function useNavigation() {
 export function useRevalidator() {
   let dataRouterContext = useDataRouterContext(DataRouterHook.UseRevalidator);
   let state = useDataRouterState(DataRouterStateHook.UseRevalidator);
+  let revalidate = React.useCallback(
+    async () => dataRouterContext.router.revalidate(),
```

Comment:
Can we keep this like it was so we don't couple the return value of `useRevalidator().revalidate` and `router.revalidate`?

```suggestion
    async revalidate() {
      await dataRouterContext.router.revalidate();
    },
```